demo-notebooks/guided-demos/notebook-ex-outputs/gptfttest.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: mcad.ibm.com/v1beta1
+apiVersion: codeflare.dev/v1beta1
```

Comment:
This should be `workload.codeflare.dev`.

For reference: https://github.com/project-codeflare/adr/blob/36041f8e6e06f9ab893b32de70f06a4593e86f09/PCF-ADR-0005-mcad-api-groups-migration.md#what